.editorconfig

```diff
@@ -0,0 +1,47 @@
+###############################################################################
+# EditorConfig is awesome: http://EditorConfig.org
+###############################################################################
```

Comment:
This isn't related to moving to repo toolset, is it?

Reply:
yes and no. I added or touched files which expect specific line endings/etc. Not having this means it is more likely to break due to a user change.